kyo-zio/shared/src/main/scala/zio/FiberInterop.scala

```diff
@@ -0,0 +1,5 @@
+package zio
```

Comment:
What do you think about creating an issue in ZIO to provide this API? I'm ok with the workaround but we could migrate to a proper API if the issue gets addressed

Reply:
I will open a PR later this week.

Reply:
https://github.com/zio/zio/pull/9246